sv.js

```diff
@@ -49,18 +49,18 @@ export default {
     roomLockedTryKnockText: 'Rummet är låst. Du kan be om att gå in genom att "knacka".',
     knockingFailedText: 'Knackningsförsöket misslyckades',
     doYouWantToKick: 'Vill du sparka ut den här användaren?',
-    deviceChanged: 'Du har anslutit eller tagit bort en input-enhet. Om det inte fungerar nu välj det gärna manuellt på inställningar.',
+    deviceChanged: 'Du har anslutit eller tagit bort en input-enhet. Om det inte fungerar, kolla gärna dina inställningar.',
     allowCameraPlz: 'Efter du trycker på OK, <b>låt</b> xroom använda din kamera.<br><br>Oroa dig inte, du ska kunna stänga av det igen innan du går in i rummet från menyn längst ner om du vill stanna osynlig.',
     tooOldBrowser: 'Din webbläsare är för gammal eller tillverkad av Microsoft. Många funktioner kan saknas.',
     pluginsLoadFailed: 'Det gick inte att ladda plugin',
     protoExpired: 'För gammal klientversion. Tryck på Ctrl&nbsp;+&nbsp;Shift&nbsp;+&nbsp;F5 för att ladda om. Om du ser detta meddelande igen, rensa webbläsarens cache manuellt tack.',
     peerError: 'Något gick fel när du utbytte data med andra. Vi föreslår att du laddar om sidan.',
-    enterpriseOnly: 'This is an enterprise only feature.',
+    enterpriseOnly: 'Det här är bara en Enterprise funktion.',
     pluginSuggested: 'Användare %s föreslog att du installerar plugin %s. Vill du göra det?',
     hostRoleRequired: 'Host-roll krävs för den här åtgärden.',
     alreadyHosted: 'Rummet har redan en värd. Gå inte in i samma rum från två webbläsarflikar. 😉',
-    roomLockedPassword: 'Rummet är låst med ett lösenord. Ange det nedan om du vet det.',
-    roomDestroyed: 'Rummet blev förstört. Du kommer att omdirigeras till startsidan.',
+    roomLockedPassword: 'Rummet är låst med ett lösenord. Ange det nedan.',
+    roomDestroyed: 'Rummet stängdes. Du kommer att omdirigeras till startsidan.',
```

Comment:
@martindomon the idea was that the rooms has actually been destroyed. I wanted to emphasize that there's no trace of it anymore in the system. "stängdes" feels like it's just closed but still exists. Maybe I'm wrong, i'm not a native speaker of course. :stuck_out_tongue:

Reply:
Haha, it's debatable. It would be the same argument for "destroyed" in english. 
You're not wrong. I just changed it based on feedback from 2 people that thought it sounded "too harsh/dangerous" 😅

It's entirely fine to keep it "förstördes".